### ISSUE TITLE
Users with small screens cannot see the bottom of the client

### ISSUE THREAD

**Issue by Pascal666** [OpenBazaar/OpenBazaar-Client#1727]
Brief Description: 
After clicking a BUY NOW button I get a box that says:
Do you have a Bitcoin Wallet?
It only takes a few minutes to set one up

There are no buttons or clickable links and nothing in the box telling me where in OpenBazaar I need to configure my wallet to get past this error screen.

Operating System (OS and version): CentOS Linux release 7.2.1511 (Core) KDE 4.14.8
OpenBazaar version (shown on `About OpenBazaar` page in menu): 1.1.6
Reproducible (Always / Almost Always / Sometimes / Rarely / Couldn't Reproduce): Always

![openbazaar wallet2](https://cloud.githubusercontent.com/assets/1330708/16365237/bc182028-3bbf-11e6-86fb-9939f9b21805.png)

**Comment by hoffmabc** [comment]
If you scroll down there should be buttons? Is your resolution old school and small?

**Comment by Pascal666** [comment]
There is no scroll bar in that dialog.  Resolution is 1280x661.  I figured out that putting an address in "Current Refund Address" in Settings will clear that error, but the next dialog box (moderated or direct) also has no scroll bar.

**Comment by jjeffryes** [comment]
The minimum height of the app window is 700px. If your monitor is cutting off the bottom of the app, then you're not going to be able to click some buttons.

You will have to click the Next buttons at the bottom of the screen to complete a purchase. Can you see them? If not, are you getting through the purchase process some other way?

**Comment by Pascal666** [comment]
No, I see no buttons and am unable to make purchases.  Everything else appears to work fine.  Is there a good reason those dialogs don't have scroll bars?

**Comment by jjeffryes** [comment]
The OpenBazaar client isn't designed for monitors smaller than 1024 x 700.

**Comment by Pascal666** [comment]
My monitor supports higher resolution, but I can't read the screen at higher resolutions.  Limiting your target audience is rarely a good idea, especially when all it would take to fix the problem is a scroll bar.

**Comment by rabfulton** [comment]
On linux at least, you can move windows higher to see thier bottom by holding alt while left clicking and dragging the window. I often have to do this when using those silly netbooks...

**Comment by jjeffryes** [comment]
I've edited the title of this issue to reflect the core problem. We should consider ways to support users with monitors below our minimum supported size.
